packages/synapse-react-client/src/components/PortalFeaturedPartners/PortalFeaturedPartners.tsx

```diff
@@ -0,0 +1,217 @@
+import { Box, Typography, CardMedia, Skeleton, Link } from '@mui/material'
+import {
+  FileHandleAssociateType,
+  FileHandleAssociation,
+  QueryBundleRequest,
+  Row,
+} from '@sage-bionetworks/synapse-types'
+import { useGetStablePresignedUrl } from '../../synapse-queries'
+import { parseEntityIdFromSqlStatement } from '../../utils/functions'
+import { SynapseConstants } from '../../utils'
+import useGetQueryResultBundle from '../../synapse-queries/entity/useGetQueryResultBundle'
+import { getFieldIndex } from '../../utils/functions/queryUtils'
+
+export type PortalFeaturedPartnersProps = {
+  sql: string
+}
+
+type Icons = {
+  partner: Row
+  imageColIndex: number
+  organizationNameColIndex: number
+  websiteColIndex: number
+  entityId: string
+  isLoading?: boolean
+}
+
+const useImageUrl = (fileId: string, entityId: string) => {
+  const fha: FileHandleAssociation = {
+    associateObjectId: entityId,
+    associateObjectType: FileHandleAssociateType.TableEntity,
+    fileHandleId: fileId || '',
+  }
+  const stablePresignedUrl = useGetStablePresignedUrl(fha, false, {
+    enabled: !!fileId,
+  })
+  const dataUrl = stablePresignedUrl?.dataUrl
+
+  return dataUrl
+}
+
+const RenderPartnerIcon = ({
+  partner,
+  imageColIndex,
+  organizationNameColIndex,
+  websiteColIndex,
+  entityId,
+  isLoading,
+}: Icons) => {
+  const fileId = partner.values[imageColIndex] ?? ''
+  const imageUrl = useImageUrl(fileId ?? '', entityId)
+
+  if (!imageUrl && !partner?.values[organizationNameColIndex]) {
+    return
+  }
+
+  return isLoading ? (
+    <Skeleton variant="rectangular" height={101} width={156} />
+  ) : (
+    <Link
```

Comment:
Remove ternary, just have a simple check for `isLoading`
```suggestion
  if (isLoading) {
    return <Skeleton variant="rectangular" height={101} width={156} />
  }
  return (
    <Link
```